scripts/drone/steps/lib.star

```diff
@@ -75,28 +75,48 @@ def identify_runner_step(platform = "linux"):
             ],
         }
 
-def clone_enterprise_step(committish = "${DRONE_COMMIT}"):
+def enterprise_setup_step(source = "${DRONE_SOURCE_BRANCH}", canFail = True):
```

Comment:
Thank you for removing `committish`! This was already driving me nuts today :D

Reply:
hahaha. I don't know if "source" is the right replacement but in the context of a PR at least it is. It should probably be `ref` or `base`.

Or `refish`? ;)